icefoganalytics/elcc-data-management
# Dev Tooling: Dockerise App so You Only Need a Single Command To Boot the Project

Relates to https://github.com/icefoganalytics/elcc-data-management/issues/3

# Context

Docker-izing the whole app so that you only need `dev boot` to boot all three containsers.

# Testing Instructions

If completely new to the project, follow the instructions in the new README.  Otherwise follow these instructions:
1. Boot the app with `dev up`
2. Go to http://localhost:8000.
3. Check that you can log in to the app. You should also be able to completely drop the database and restore it with the instructions in the README.
